device/iot-device-samples/pnp-device-sample/temperature-controller-device-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/device/TemperatureController.java

```diff
@@ -402,6 +403,59 @@ public void TwinPropertyCallBack(Property property, Object context) {
         }
     }
 
+    // Report the property updates on "deviceInformation" component.
+    private static void updateDeviceInformation() throws IOException {
+        String componentName = "deviceInformation";
+
+        String manufacturer = "manufacturer";
```

Comment:
No, this has to be changed to update all the properties in 1 go. Stefan said this was unacceptable WRT message explosion. See my .NET helpers update.

Reply:
I'll open another PR to fix that then